web/web-mock/web-mock-soap/src/main/java/com/castlemock/web/mock/soap/controller/mock/AbstractSoapServiceController.java

```diff
@@ -148,36 +141,34 @@ protected AbstractSoapServiceController(final ServiceProcessor serviceProcessor,
         try{
             Preconditions.checkNotNull(projectId, "THe project id cannot be null");
             Preconditions.checkNotNull(httpServletRequest, "The HTTP Servlet Request cannot be null");
-            Enumeration<String> parameterNames = httpServletRequest.getParameterNames();
-            while(parameterNames.hasMoreElements()){
-                String parameterName = parameterNames.nextElement();
-                if(parameterName.equalsIgnoreCase("wsdl")){
-                    String wsdl = getWsdl(projectId);
+            String requestUri = httpServletRequest.getRequestURI();
```

Comment:
Minor thing as well, but neither the requestUri or wsdl are being updated, so they could be marked as final.